### PR TITLE
fix:Refresh tooltip content

### PR DESCRIPTION
**Pull request by yang1666204**
## Summary
1. The topology map resource pool is updated to tooltip after editing.
2. Solve the problem of occasional white screen when editing resource pool.
3. The main tenant synchronization function in the created tenant filters out the running tenants.
4. Solve some UI issues.

### CODE REVIEW

File: ui/src/pages/Tenant/New/TenantSource.tsx [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
-  DatePicker,
-  Form,
-  Input,
-  Row,
-  Select,
-  Space,
-  Switch,
-  TimePicker,
-  Tooltip,
+Card,
```
Comment (powerfooI) [inline ui/src/pages/Tenant/New/TenantSource.tsx:5]: Why do you remove leading spaces here?

File: ui/src/components/TopoComponent/helper.ts [hunk 2]
```diff
@@ -233,6 +233,22 @@ export const checkIsSame = (oldTopoData: any, newTopoData: any): boolean => {
   return true;
 };
 
+// export const checkReplicasIsSame = (
```
Comment (powerfooI) [inline ui/src/components/TopoComponent/helper.ts:236]: Remove this if it's unused.

**Review by powerfooI — CHANGES_REQUESTED** [review]

File: ui/src/pages/Tenant/New/TenantSource.tsx (hunk 1 above)
Comment (yang1666204) [reply ui/src/pages/Tenant/New/TenantSource.tsx:5]: The internationalization tool automatically formats.

Comment (yang1666204) [reply ui/src/pages/Tenant/New/TenantSource.tsx:5]: But I can modify the configuration of the tool so that it formats the code in the ideal way

**Review by powerfooI — APPROVED** [review]
LGTM

**Pull request merged**